modules/library/metadata/src/main/java/org/geotools/measure/Measure.java

```diff
@@ -112,10 +112,8 @@ public boolean equals(final Object object) {
     /** Returns a string representation of this measure. */
     @Override
     public String toString() {
-        final StringBuilder buffer = new StringBuilder();
-        buffer.append(value);
-        buffer.append(' ');
-        buffer.append(unit);
-        return buffer.toString();
+        return String.valueOf(value)
+                + ' '
+                + GeoToolsUnitFormatterFactory.getUnitFormatterSingleton().format(unit);
```

Comment:
As you can see, the GeoTools project chose to use the Google AOSP formatter. That limits the code to 80 columns, it implies identifiers should be clear but short.

Something like ``UnitFormatter.getInstance()`` would be clear enough while fitting better in the line length limitations:
- The GeoTools bit is provided by the import
- The class is an atypical factory, in that it's just a holder for a singleton, so no need to stress the factory aspect in the name
- The "getInstance" method is normally used in singletons to grab the instance, it's evocative enough.

The other formatter factories should be renamed accordingly.

Since BaseUnitFormatter is not final, it's also possible to rewrite the "factory" as a true singleton. But.. your choice :-D

Reply:
`GeoToolsUnitFormatterFactory` equivalent was `GeoToolsUnitFormat` before ... UnitFormatter would clash with the UnitFormatter interface though, `Default` would be confusing because that was used for a different unit formatter before, we already use `Base` ... `Standard` would also be difficult, because we adding all kinds of non-standard units ...

Reply:
Both these alternatives:

```
GTUnitFormatter.getInstance()
GeoToolsUnitFormatter.getInstance()
```

would still be significantly shorter than:

```
GeoToolsUnitFormatterFactory.getUnitFormatterSingleton()
```

Try them and see how many line wraps they spare.

Reply:
Shuffled names around, hopefully this is better.